src/applications/app-errors.js

```diff
@@ -14,15 +14,15 @@ export function handleAppError(err, app) {
 
 export function addErrorHandler(handler) {
   if (typeof handler !== 'function') {
-    throw new Error('a single-spa error handler must be a function');
+    throw Error('a single-spa error handler must be a function');
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Used_as_a_function

This saves FOUR WHOLE BYTES every time we get rid of `new ` :D :D 

Not super important, but I decided to make them all consistent, at least.